stix_shifter_modules/security_advisor/stix_transmission/auth.py

```diff
@@ -25,4 +26,15 @@ def obtainAccessToken(self):
             return response.json()["access_token"]
 
         else:
-            raise Exception("Authorizaion Failed")
\ No newline at end of file
+            raise Exception("Authorizaion Failed")
+
+    def find_location(self, accountID, host):
+        if host == "":
+            adminApiUrl = os.getenv('ADMIN_API_URL', 'https://compliance.cloud.ibm.com/admin/v1')
```

Comment:
You may want to define `https://compliance.cloud.ibm.com/admin/v1` in a constant at the top just so it's easier to find if that default API url changes in the future.

Reply:
Done